composer.json

```diff
@@ -93,13 +93,13 @@
     ],
 
     "require": {
-        "php":                          "^5.3.3|^7.0",
-        "symfony/framework-bundle":     "^2.3|^3.0",
-        "symfony/security-bundle":      "^2.3|^3.0",
-        "symfony/options-resolver":     "^2.3|^3.0",
-        "symfony/form":                 "^2.3|^3.0",
-        "symfony/yaml":                 "^2.3|^3.0",
-        "symfony/templating":           "^2.7|^3.0",
+        "php":                          "^5.6|^7.0",
+        "symfony/framework-bundle":     "^2.8|^3.0",
```

Comment:
IMO this should be still LTS `2.7` version as it will be supported same way as `2.8`.